app/layouts/default-layout.tsx

```diff
@@ -153,7 +153,7 @@ const DefaultLayout: React.FunctionComponent<Props> = ({
 
           @media screen and (min-width: 992px) {
             #page-content {
-              padding-top: 110px;
+              padding-top: 60px;
```

Comment:
This should have one exception so we can keep the existing top padding on the homepage. You can do that by adding the following rule in `pages/index`. (before this can work you'll need to replace the first `<Row>` with `<div className="row">`):

```
          @media screen and (min-width: 992px) {
            .row:first-child {
              padding-top: 50px;
            }
          }
```

Reply:
why do we want to keep the existing 110px padding on the home page?

Reply:
It's for aesthetic reasons: The current whitespace balance creates a centering effect - thus emphasis - on the welcome content (in addition to consistency).

The whitespace shown here between the 3 main content blocks repeats throughout this page, including between the lower content and the footer, and between the top block and the header. The 60px default changes that and creates a mild crowding effect by squishing the top content block up into the header.
 
Current layout:
<img width="1440" alt="Screen Shot 2021-03-22 at 12 40 59 PM" src="https://user-images.githubusercontent.com/5522075/112049048-5a8d2500-8b0c-11eb-9173-9671bf0ee095.png">
currently lets the

Reply:
(in other words, I suspect the original 110px setting may have been introduced specifically to balance the homepage, but unintentionally applied to the whole site via the default template).